<launch>                                                                        
  <node pkg="cv_camera" type="cv_camera_node" name="cv_camera_node1" >
    <param name="device_id" value="0" />
    <param name="image_height" value="600" />
    <param name="image_width" value="800" />
  </node>


  <node name="laser_pointer_tracker1" pkg="laser_pointer_tracker" type="laser_tracker.py" respawn="false" output="screen">
    <param name="input_image" value="cv_camera_node1/image_raw" />
    <param name="hue_min" value="20" />
    <param name="hue_max" value="160" />
    <param name="sat_min" value="100" />
    <param name="sat_max" value="255" />
    <param name="val_min" value="200" />
    <param name="val_max" value="256" />
    <param name="radius_min" value="2" />
    <param name="radius_max" value="100" />
    <param name="enable_cal_out" value="True" />
  </node>"
</launch>